src/main/java/nc/students/ayaz/model/Comment.java

```diff
@@ -1,13 +1,24 @@
-package nc.students.ayaz;
+package nc.students.ayaz.model;
 
 import cobertura.IgnoreDuringCodeCoverage;
 
-class Comment {
+public class Comment {
 
     private String author;
+
     private String text;
 
-    Comment(String author, String text) {
+    @IgnoreDuringCodeCoverage
+    public String getAuthor() {
```

Comment:
There are two distinct cases (people associate with getters/setters):
1) A member field has a getter and a setter. Its an anti-pattern. People probably do it because they don't want to make their member field "public" and they try (in vain, though) hide it behind getter/setter. This is stupid, if you need read and write your field outside your class you should just make it public. Better yet, to think again and check your design, because you probably have a design problem if you need to access a field from other class in a read/write manner.
2) A member field has either a getter or a setter. Then such a method shouldn't actually be considered as being in getter/setter category. Such a method is just one of many other public methods that comprise a public interface for your class. And you always want to make sure that a class properly implements its interface. So, you do it by testing it.